encoding/protobuf_test.go

```diff
@@ -188,3 +189,40 @@ func TestProtobufResponse(t *testing.T) {
 		})
 	}
 }
+
+func Test_anyResolver_Resolve(t *testing.T) {
+	source, err := protobuf.NewDescriptorProviderFileDescriptorSetBins("../testdata/protobuf/simple/simple.proto.bin")
+	assert.NoError(t, err)
+	tests := []struct {
+		name    string
+		typeUrl string
+		resolveType interface{}
+		wantErr bool
```

Comment:
There is no error testcase, this should either be removed or preferably an error case should be added.